src/replication/replicator.rst

```diff
@@ -154,98 +203,253 @@ the following order:
+   non-continuous replications. Once in this state the replication is
+   "forgotten" by the scheduler and it doesn't consume any more CPU or
+   memory resorces. Continuous replication jobs will never reach this
+   state.
+
+.. _Normal vs Continuous Replications:
+
+Normal vs Continuous Replications
+---------------------------------
+
+Normal (non-continuous) replications once started will be allowed to
+run to completion. That behavior is to preserve their semantics of
+replicating a snapshot of the source database to the target. For
+example if new documents are added to the source after the replication
+are started, those updates should not show up on the target database.
+Stopping and restring a normal replication would violate that
+constraint.
+
+.. warning:: When there is a mix of continuous and normal
+    replications, once normal replication are scheduled to run, they
+    might temporarily starve continuous replication jobs.
+
+However, normal replications will still be stopped and rescheduled if
+an operator reduces the value for the maximum number of replications.
+This is so that if an operator decides replications are overwhelming a
+node that it has the ability to recover. Any stopped replications will
+be resubmitted to the queue to be rescheduled.
+
+Compatibility Mode
+==================
+
+Previous version of CouchDB replicator wrote state updates back to
+replication documents. In case were user code programmatically read
+those states, there is compatibility mode enabled via a configuration
+setting::
+
+  [replicator]
+  update_docs = true
+
+In this mode replicator will continue to write state updates to the
+documents.
+
+To effectively disable the scheduling behavior, which periodically
+stop and starts jobs, set ``max_jobs`` configuration setting to a
+large number. For example::
+
+  [replicator]
+  max_jobs = 9999999
+
+See :ref:`Replicator configuration section <config/replicator>` for
+other replicator configuration options.
 
 Canceling replications
 ======================
 
-To cancel a replication simply ``DELETE`` the document which triggered the
-replication. The Couch log will show you an entry like the following:
-
-.. code-block:: text
-
-    [Thu, 17 Feb 2011 20:16:29 GMT] [info] [<0.125.0>] Stopped replication `c0ebe9256695ff083347cbf95f93e280+continuous+create_target` because replication document `doc_A` was deleted
-
-.. note::
-    You need to ``DELETE`` the document that triggered the replication.
-    ``DELETE``-ing another document that describes the same replication
-    but did not trigger it, will not cancel the replication.
+To cancel a replication simply ``DELETE`` the document which triggered
+the replication. To update a replication, for example, change the
+number of worker or the source, simply update the document with new
+data. If there is extra application-specific data in the replication
+documents, that data is ignored by the replicator.
```

Comment:
Is there a way to cancel a `_replicate` triggered replication? If so I'd imagine it's via the new endpoint, and you probably want to explicitly show an example. Otherwise note that `_replicate` triggered replications cannot be cancelled.